src/core/annotation.js

```diff
@@ -2825,8 +2809,8 @@ class ButtonWidgetAnnotation extends WidgetAnnotation {
       return null;
     }
     const storageEntry = annotationStorage.get(this.data.id);
```

Comment:
Same here, this likely also requires a `?` addition.

Reply:
Just above we have an early return for `!annotationStorage`, hence that shouldn't be necessary in this case.